code/extensions/che-api/tests/github-service-impl.spec.ts

```diff
@@ -12,59 +12,83 @@
 import 'reflect-metadata';
```

Comment:
unfortunately tests are not integrated in the build process
so, I would say they are run never: https://github.com/eclipse/che/issues/22213
the only way to run them at the moment - manually...